CleverPush/Source/CleverPushInstance.m

```diff
@@ -1129,6 +1128,13 @@ - (void)subscribe:(CPHandleSubscribedBlock _Nullable)subscribedBlock failure:(CP
                                         }
                                     }];
                                 }
+
+                                if (successBlockCalled) {
```

Comment:
That will not work as the code above executes async code. It means that `successBlockCalled` might be set to `YES` after this code here executes, as this is in an async callback:
https://github.com/cleverpush/cleverpush-ios-sdk/pull/202/files#diff-f5326f84792597a7145e9b2f5be8823e3bc651f5b4faac02dc05dca951eee90fR1122